packages/knip/src/plugins/jest/index.ts

```diff
@@ -58,7 +58,14 @@ const resolveDependencies = async (config: JestInitialOptions, options: PluginOp
 
   const runner = config.runner ? [config.runner] : [];
   const runtime = config.runtime && config.runtime !== 'jest-circus' ? [config.runtime] : [];
-  const environments = config.testEnvironment === 'jsdom' ? ['jest-environment-jsdom'] : [];
+
+  let environments: Array<string> = [];
+  if (config.testEnvironment === 'jsdom') {
+    environments = ['jest-environment-jsdom'];
+  } else if (config.testEnvironment) {
+    environments = [config.testEnvironment];
+  }
```

Comment:
Maybe we could make it a `const` in one line? That's become a bit of a standard whenever possible, for consistency.

Reply:
It would require a nested ternary but I can do that!

Reply:
fixed in https://github.com/webpro-nl/knip/pull/834/commits/b80e392ae1433677a9331e92c2a09837323623e1